jung-visualization/src/main/java/edu/uci/ics/jung/visualization/transform/LayoutLensSupport.java

```diff
@@ -30,12 +32,14 @@
   private static final Logger log = LoggerFactory.getLogger(LayoutLensSupport.class);
   protected NetworkElementAccessor<V, E> pickSupport;
 
-  public LayoutLensSupport(VisualizationViewer<V, E> vv, Dimension d) {
-    this(
-        vv,
+  public LayoutLensSupport(VisualizationViewer<V, E> vv) {
```

Comment:
I don't see any signs that this constructor, or the one below, are setting the `pickSupport`.  Also, the relationship between this constructor and the one that takes a VV and a `Lens` is kind of confusing: why isn't this constructor calling that one?

Reply:
good catch on the pickSupport.
On the this() call, I was thinking the inline creation of the Lens would be really ugly, but I can use model.getLayoutSize() which i am very torn about leaving in place. I'll do that for now though